src/app/meetups/meetups-add.component.html

```diff
@@ -0,0 +1,27 @@
+<mat-toolbar>
+  <a mat-icon-button routerLink="/meetups">
+    <mat-icon>arrow_back</mat-icon>
+  </a>
+</mat-toolbar>
+
+<div class="space-container">
+  <mat-toolbar class="primary-color font-size-1" i18n>New Meetup</mat-toolbar>
+  <div class="view-container">
+    <form class="view-container" [formGroup]="meetupForm" (ngSubmit)="onSubmit()" novalidate>
+      <mat-form-field>
```

Comment:
Sorry, looks like we should have block elements surrounding these form fields, so please revert to having these two `mat-form-fields` surrounded by `div` elements